### PR TITLE
Utils: added link to API classes, update homepage

### PR DESCRIPTION
**Pull request by f3l1x**
I've added link to main Utils page and also I've added links to API classes on that page.

### CODE REVIEW

File: cs/homepage.texy [hunk 1]
```diff
@@ -66,7 +66,7 @@
 - [Vyhledávání souborů a adresářů | finder]
 - [Atomické operace | atomicity]
 - [Validace | validators]
-- Funkce pro práci s [řetězci | strings], [poli | arrays], [HTML elementy | HTML elements], [datem | datetime] a [URL | URLs]
+- [Utility | utils] (funkce pro práci s [řetězci | strings], [poli | arrays], [HTML elementy | HTML elements], [datem | datetime] a [URL | URLs])
```
Comment (dg) [inline cs/homepage.texy]: great!

File: cs/utils.texy [hunk 2]
```diff
@@ -4,12 +4,22 @@ Utility
...
+=========
+
+/--
+composer require nette/utils
+\--
+
+Užitečné třídy
+==============
+
+| [Rozšíření objektů v PHP | PHP Language Enhancements] | [Nette\SmartObject |api:]
```
Comment (dg) [inline cs/utils.texy]: I think it makes the table unclear, ie. it is not entirely clear which link to actually click on.

Comment (f3l1x) [inline cs/utils.texy]: But how do you get to detail page? What about convert table to headlines with short description? Wouldn't be better?

Comment (dg) [inline cs/utils.texy]: Link to detail page is IMHO enough.

Comment (f3l1x) [inline cs/utils.texy]: It should be better now.

File: en/utils.texy [hunk 3]
```diff
@@ -4,6 +4,16 @@ Utility
 .[perex]
 In package `nette/utils` you will find a set of useful classes for everyday use:
 
+Instalation
```
Comment (dg) [inline en/utils.texy]: Installation with two l

Comment (f3l1x) [inline en/utils.texy]: Oh, sorry. Solved.

**Comment by dg** [comment]
Thanks!

**Comment by f3l1x** [comment]
Pleasure is on my side.

**Pull request merged**